src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/MulticlassLogisticRegression.cs

```diff
@@ -908,46 +866,46 @@ bool ISingleCanSaveOnnx.SaveAsOnnx(OnnxContext ctx, string[] outputs, string fea
         /// of <paramref name="weights"/>.</param>
         public void GetWeights(ref VBuffer<float>[] weights, out int numClasses)
         {
-            numClasses = _numClasses;
-            Utils.EnsureSize(ref weights, _numClasses, _numClasses);
-            for (int i = 0; i < _numClasses; i++)
-                _weights[i].CopyTo(ref weights[i]);
+            numClasses = NumberOfClasses;
+            Utils.EnsureSize(ref weights, NumberOfClasses, NumberOfClasses);
+            for (int i = 0; i < NumberOfClasses; i++)
+                Weights[i].CopyTo(ref weights[i]);
+        }
+
+        /// <summary>
+        /// Gets the biases for the logistic regression predictor.
+        /// </summary>
+        public IEnumerable<float> GetBiases()
+        {
+            return Biases;
         }
 
         internal IEnumerable<float> DenseWeightsEnumerable()
```

Comment:
Not exactly related to your PR, but I found it strange what we expose weights and Bias(es) differently for binary and multiclass model.

Reply:
The reason behind is that those parameters got one extra dimension. In binary, the bias is a scalar but in multiclass, the bias is a vector with #-of-class elements.

---
In reply to: [266661678](https://github.com/dotnet/machinelearning/pull/2976#discussion_r266661678) [](ancestors = 266661678)

Reply:
Which is weird.
NumClasses should be property of the model, bias and weights should be IReadonly<float> etc, but all this is out of scope of this PR:)

---
In reply to: [266668443](https://github.com/dotnet/machinelearning/pull/2976#discussion_r266668443) [](ancestors = 266668443,266661678)